docs/assemblies/assembly-prometheus-metrics-proxy.adoc

```diff
@@ -0,0 +1,79 @@
+// file included in the following:
+//
+// kroxylicious-operator/index.adoc
+
+[id='assembly-prometheus-metrics-proxy-{context}']
+= Proxy Metrics
+
+[role="_abstract"]
+
+The proxy provides *connections metrics* and *message metrics*.
+
+Connection metrics allow the connections made from the clients to the proxy (the downstream) and the connection made by the proxy to the kafka cluster (the upstream) to be understood.
+
+Message metrics allow the kafka protocol messages that flow through the proxy to be understood.
+
+== Connection Metrics
+
+The connection metrics counts connection made to the proxy from the clients, and by the proxy to the brokers.
+
+Connection metrics are discriminated by the virtual cluster's name and the broker's node ID.  When the client
+connects to boostrap a node ID value of `bootstrap` is recorded.
+
+|===
+|Metric Name |Type |Labels|Description
+
+|kroxylicious_client_to_proxy_connection_total
+|Counter
+|`virtual_cluster`, `node_id`
+|Incremented by one every time a connection is accepted from the downstream by the proxy.
+
+|kroxylicious_client_to_proxy_errors_total
+|Counter
+|`virtual_cluster`, `node_id`
+|Incremented by one every time a connection is closed due to any downstream error.
+
+|kroxylicious_proxy_to_server_connections_total
+|Counter
+|`virtual_cluster`, `node_id`
+|Incremented by one every time a connection is made to the upstream from the proxy.
+
+|kroxylicious_proxy_to_server_errors_total
+|Counter
+|`virtual_cluster`, `node_id`
+|Incremented by one every time a connection is closed due to any upstream error. #3
+|===
+
+== Message Metrics
+
+The message metrics count the kafka protocol messages that flow through the proxy.  The metrics
```

Comment:
We probably aren't consistent in the code either, but we often refer to them as RPCs rather than messages. Is that a more precise/appropriate term to use in our docs?